<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingWidget</class>
 <widget class="QWidget" name="TrainingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>_(&quot;Rack:&quot;)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditRack">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>_(&quot;Play a word:&quot;)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPlay">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>_(&quot;Ref.:&quot;)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCoords">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>_(&quot;Enter the coordinates of the word. Specify the row before the column for horizontal words, and the column before the row for vertical words. E.g.: H4 or 4H&quot;)</string>
       </property>
       <property name="statusTip">
        <string>_(&quot;Enter the coordinates of the word. Specify the row before the column for horizontal words, and the column before the row for vertical words. E.g.: H4 or 4H&quot;)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>_(&quot;Points:&quot;)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPoints">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPlay">
       <property name="text">
        <string>_(&quot;Play&quot;)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="pushButtonRack">
       <property name="text">
        <string>_(&quot;New rack&quot;)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonComplement">
       <property name="text">
        <string>_(&quot;Complement&quot;)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSearch">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>_(&quot;Search&quot;)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPlaySelected">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>_(&quot;Play selected&quot;)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>_(&quot;Search results&quot;)</string>
     </property>
     <layout class="QVBoxLayout" name="_2">
      <item>
       <widget class="QTreeView" name="treeViewResults">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="indentation">
         <number>0</number>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditRack</tabstop>
  <tabstop>lineEditPlay</tabstop>
  <tabstop>lineEditCoords</tabstop>
  <tabstop>lineEditPoints</tabstop>
  <tabstop>pushButtonPlay</tabstop>
  <tabstop>pushButtonRack</tabstop>
  <tabstop>pushButtonComplement</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonPlaySelected</tabstop>
  <tabstop>treeViewResults</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditRack</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtonSearch</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
